Minio.Examples/Cases/ListObjects.cs

```diff
@@ -25,18 +25,21 @@ internal static class ListObjects
         string bucketName = "my-bucket-name",
         string prefix = null,
         bool recursive = true,
-        bool versions = false)
+        bool versions = false,
+        bool includeUserMetadata = false)
     {
         try
         {
             Console.WriteLine("Running example for API: ListObjectsAsync");
             var listArgs = new ListObjectsArgs()
                 .WithBucket(bucketName)
                 .WithPrefix(prefix)
-                .WithRecursive(recursive);
+                .WithRecursive(recursive)
+                .WithVersions(versions)
+                .WithUserMetadata(includeUserMetadata);
             var observable = minio.ListObjectsAsync(listArgs);
             var subscription = observable.Subscribe(
```

Comment:
This method returns directly after the `Subscribe`, so it will run in parallel with other tests and that will make it prone to errors. Better use something like this
```cs
            var tcs = new TaskCompletionSource();
            using var subscription = observable.Subscribe(
                item => Console.WriteLine($"Object: {item.Key}, content-type: {item.UserMetadata.ToList()[0].Value}"),
                ex => tcs.SetException(ex),
                () => tcs.SetResult());
            await tcs.Task.ConfigureAwait(false);
```
That will convert the subscription into a task and you can properly wait for it. This problem was already in the code before your PR, but it makes it hard to debug.

Reply:
This fix can be in a separate PR as it is completely unrelated.
I'll create a separate issue/PR for it.

Reply:
If listing objects in the SDK doesn't work correctly, then I don't think we should even release it.

Reply:
@ramondeklein ,
Done.
Please review it again.